Header header
float64 time_total # arm cylinder pressure (large chamber)
float64 time_track # arm cylinder pressure (small chamber)
float64 time_1p    # swing pressure (left)
float64 time_5p    # swing pressure (right)
float64 time_new   # swing pressure (right)

float64 n_initial
float64 n_pass_bidirection
float64 n_pass_1p
float64 n_pass_5p
float64 n_new
float64 n_final
float64 n_ok_parallax

float64 avg_parallax
float64 avg_age

float64 steering_angle
